### DISCUSSION TITLE
How can I generate sentence in English

### DISCUSSION THREAD

**Discussion by eugenia4it** [bchavez/Bogus#386] [Q&A] [answered]
I followed the example to generate sentence in English. However they are not English. Can you please tell me how can I generate in en_US? Thanks a lot.
```
public async static Task Main(string[] args)
        {
            var faker = new Faker("en");
            var item = faker.Lorem.Sentence();
            var item1 = new Bogus.DataSets.Lorem(locale: "en_US");

            Console.WriteLine(item);
            Console.WriteLine(item1.Sentence(5));
        }
```
Result 1:   
Eveniet aspernatur deserunt aut debitis.
Aut dicta ut ad aut.
Result 2:
Eos dicta illum maiores quo veniam ratione est consectetur.
Voluptates reprehenderit quas adipisci eos.

**Comment by bchavez** [comment, marked as answer]
Bogus does not have artificial intelligence and cannot generate perfectly grammatically correct English sentences on its own in any thoughtful coherent way. So, you have a couple of options:

1. Use [**WaffelGenerator.Bogus** extension](https://github.com/SimonCropp/WaffleGenerator#wafflegeneratorbogus) which _tries_ to generate better "English" sentences
2. Get a large chunk of text, like the [US Constitution](https://www.usconstitution.net/const.txt) or some other [free eBook](https://www.gutenberg.org) and parse sentences from them.
3. Use a method in `Bogus.Text` in [**Bogus Premium**](https://github.com/bchavez/Bogus/wiki/Bogus-Premium) that already does (option 2) for you.

**Reply by eugenia4it** [reply]
Thank you! I appreciate your answers. It's very helpful :)
